api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -385,6 +388,31 @@ class Config:
         orm_mode = True
 
 
+class IndividualOfferResponseGetterDict(GetterDict):
+    def get(self, key: str, default: Any | None = None) -> Any:
+        if key == "address":
+            if FeatureToggle.WIP_ENABLE_OFFERER_ADDRESS.is_active():
+                if self._obj.offererAddress is not None:
```

Comment:
```suggestion
            if not FeatureToggle.WIP_ENABLE_OFFERER_ADDRESS.is_active() or self._obj.offererAddress is None:
                return None
            
            offererAddress = ...
```
(détail) Je suis pour les early return qui permettent réduire le nombre d'indentations, je trouve ça plus lisible.